script.js

```diff
@@ -0,0 +1,89 @@
+import Catalogo from "./catalogo.js";
+import Auto from "./auto.js";
+
+let catalogo = new Catalogo();
+
+let opcion;
+
+do {
+  opcion = prompt(
+    "Menú:\n" +
+      "1. Agregar auto\n" +
+      "2. Mostrar catálogo\n" +
+      "3. Ordenar autos por precio\n" +
+      "4. Filtrar por marca\n" +
+      "5. Buscar auto por modelo\n" +
+      "6. Salir (Si es primera vez, salir y actualizar)\n" +
+      "Ingrese una opción:"
+  );
+
+  switch (opcion) {
+    case "1":
+      const marca = prompt("Ingrese la marca del auto:");
+      const modelo = prompt("Ingrese el modelo del auto:");
+      const anio = prompt("Ingrese el año del auto:");
+      const kilometros = prompt("Ingrese los kilómetros recorridos:");
+      const precio = prompt("Ingrese el precio del auto:");
+      const id = catalogo.autos.length + 1;
```

Comment:
Esto lo podria hacer dentro de Catalogo, en `agregarAuto()`
```suggestion
agregarAuto(marca, modelo) {
    const id = this.autos.length + 1; // o usar un contador interno mejor
    const nuevoAuto = new Auto(id, marca, modelo);
    this.autos.push(nuevoAuto);
  }
  ```